connectors/src/connectors/notion/temporal/activities.ts

```diff
@@ -758,16 +758,18 @@ export async function garbageCollect({
     // Temporary performance logging.
     const startTime = performance.now();
 
+    // Find the resources not seen in the GC run (using runTimestamp).
     resourcesToCheck = await findResourcesNotSeenInGarbageCollectionRun(
       connector.id,
-      startTs
+      runTimestamp
```

Comment:
We should not use `startTs` here but rather `runTimestamp` which is used here:

https://github.com/dust-tt/dust/blob/dda198820606bf72677cdfc67d4216893597f5b8/connectors/src/connectors/notion/temporal/activities.ts#L865-L897